docs/openapi/components/schemas/common/AgGradeInspectionLot.yml

```diff
@@ -0,0 +1,255 @@
+$linkedData:
+  term: AgGradeInspectionLot
+  '@id': https://w3id.org/traceability#AgGradeInspectionLot
+title: Agriculture Grade Inspection Lot
+description: Information regarding the inspected lot including samples, defects and grades.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - AgGradeInspectionLot
+      - type: string
+        const:
+          - AgInspectionLotesult
+  agProduct:
+    title: Product
+    description: The product inspected.
+    $ref: ./AgProduct.yml
+    $linkedData:
+      term: agProduct
+      '@id': https://w3id.org/traceability#AgProduct
+  lotIdentifier:
+    title: Lot Identifier
+    description: Use to further add to identification of the lot inspected, or to distinguish between lots / sublots for certification purposes (e.g. "Loaded", "Main Lot", "U.S. Fancy", "Film Wrapped").
+    type: string
+    $linkedData:
+      term: lotIdentifier
+      '@id': https://schema.org/identifier
```

Comment:
GS1 maybe?

Reply:
Thanks, that seems like a better place to look!

The closest I'm seeing is `gs1.org/voc/hasBatchLotNumber`. I'm a little confused, though, because while Lot Identifier seems like it'd be a good fit, I'm only seeing the gs1 property `hasBatchLotNumber`, which as far as I can tell describes a relationship between `Product` and `string` rather than a class?

Could you provide some guidance? Otherwise I'll just go with `gs1.org/voc/hasBatchLotNumber`, as there seems to be some precedence for using properties (e.g. `gs1.org/voc/variantDescription` on `Package.yml`)